pages/linux/faillock.md

```diff
@@ -0,0 +1,16 @@
+# faillock
+
+> Display and modify authentication failure record files.
+> More information: <https://manned.org/faillock>.
+
+- List login failures of all users:
+
+`sudo faillock`
+
+- List login failures for the specified user:
```

Comment:
```suggestion
- List login failures of the specified user:
```

Personally I prefer of here @mfrw and i think we have used it more commonly in this context. What do you think, regardless they mean the same thing.

Reply:
Slight "of" preference. Accepting, considering it has thumbs up from @mfrw.